src/globus_cli/commands/streams/tunnel/list.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import command
+from globus_cli.termio import Field, display
+
+TUNNEL_LIST_FIELDS = [
+    Field("ID", "id"),
+    Field("Label", "attributes.label"),
+    Field("State", "attributes.state"),
+    Field("Created Time", "attributes.created_time"),
+]
+
+
+@command("list", short_help="List Globus Tunnels.")
+@LoginManager.requires_login("auth", "transfer")
+def list_tunnel_command(
+    login_manager: LoginManager,
+) -> None:
+    """
+    List all the Globus Tunnels that this user can see.
```

Comment:
Tunnel (as a name for the resource) appears in uppercase and lowercase in this PR. Recommending lowercase for consistency.


```suggestion
    List all the Globus tunnels that this user can see.
```